products/sle/main.pm

```diff
@@ -1033,6 +1033,7 @@ else {
         loadtest "migration/post_upgrade";
     }
     elsif (get_var("AUTOYAST") || get_var("AUTOUPGRADE")) {
+        # We use startshell to syn reboot on s390x
```

Comment:
what do you mean with "syn reboot"?

Reply:
This comment is irrelevant anyway, I'll remove it. It was supposed to be "sync" =) Thanks!